relay-event-normalization/src/event.rs

```diff
@@ -144,6 +144,9 @@ pub fn normalize_event(event: &mut Annotated<Event>, config: &NormalizationConfi
     let _ = legacy::LegacyProcessor.process_event(event, meta, ProcessingState::root());
 
     if !is_renormalize {
+        // Check for required and non-empty values
+        let _ = schema::SchemaProcessor.process_event(event, meta, ProcessingState::root());
```

Comment:
And this is fine to run the schema processor before transactions processor which wasn't the case before this change?

Reply:
The schema processor doesn't perform any normalization, all it does is add errors to the payload's meta when required attributes are missing or have invalid values.